src/postgres/abstract/models.ts

```diff
@@ -47,7 +47,7 @@ export interface AbstractServerCreate {
     sku: AbstractSku;
     administratorLogin: string;
     administratorLoginPassword: string;
-    storageMB: number;
+    storageGB: number;
```

Comment:
Flexible servers now need their storage size in GB instead of MB. I had to make a few changes to accommodate this.